docs/docs/dev_docs/getting_started/cli.md

```diff
@@ -24,10 +24,22 @@ Or if you use yarn:
 yarn global add @aztec/cli
 ```
 
-Then verify that it is installed with:
+If you have specified a SANDBOX_VERSION environmental variable when installing sandbox, you will need to specify the same version when installing the CLI. For example:
```

Comment:
Please could you also include an instruction here for how a user can query their current version of the sandbox? (In case the `$SANDBOX_VERSION` environment variable isn't set (e.g. a new shell))